site/_utils/drafts.js

```diff
@@ -5,12 +5,11 @@
  * @param {EleventyCollectionItem} item
  * @return {boolean}
  */
-const drafts = item => {
-  if (process.env.NODE_ENV !== 'production') {
-    return true;
-  } else {
+const filterDrafts = item => {
+  if (process.env.NODE_ENV === 'production') {
     return !item.data.draft;
   }
+  return true; // include everything in non-prod
 };
 
-module.exports = {drafts};
+module.exports = {filterDrafts};
```

Comment:
I'd suggest filterOutDrafts or removeDrafts. Also, this looks like an eleventy filter, should it be in site/_filters?  How does this compare to _filters/removeDrafts?

Reply:
It's not really an 11ty filter, which is confusing. It's just a helper for collection filtering. I've made that clearer, and I like the updated name—thanks.